PerforceServerSetupTemplate.yaml failing at 14_create_symlink

From the `cfn-init.log`:
```
...
2021-02-16 15:07:21,118 [INFO] Command 13_rewrite_config succeeded
2021-02-16 15:07:21,119 [DEBUG] Command 13_rewrite_config output:
2021-02-16 15:07:21,119 [DEBUG] Running command 14_create_symlink
2021-02-16 15:07:21,119 [DEBUG] No test for command 14_create_symlink
2021-02-16 15:07:21,122 [ERROR] Command 14_create_symlink (ln -s /opt/perforce/bin/p4 /hxdepots/sdp/Server/Unix/p4/common/bin/p4) failed
2021-02-16 15:07:21,122 [DEBUG] Command 14_create_symlink output: ln: failed to create symbolic link '/hxdepots/sdp/Server/Unix/p4/common/bin/p4': File exists

2021-02-16 15:07:21,122 [ERROR] Error encountered during build of 04_config_install_sdp: Command 14_create_symlink failed
Traceback (most recent call last):
  File "/usr/lib/python2.7/site-packages/cfnbootstrap/construction.py", line 542, in run_config
    CloudFormationCarpenter(config, self._auth_config).build(worklog)
  File "/usr/lib/python2.7/site-packages/cfnbootstrap/construction.py", line 260, in build
    changes['commands'] = CommandTool().apply(self._config.commands)
  File "/usr/lib/python2.7/site-packages/cfnbootstrap/command_tool.py", line 117, in apply
    raise ToolError(u"Command %s failed" % name)
ToolError: Command 14_create_symlink failed
2021-02-16 15:07:21,124 [ERROR] -----------------------BUILD FAILED!------------------------
2021-02-16 15:07:21,124 [ERROR] Unhandled exception during build: Command 14_create_symlink failed
Traceback (most recent call last):
  File "/opt/aws/bin/cfn-init", line 171, in <module>
    worklog.build(metadata, configSets)
  File "/usr/lib/python2.7/site-packages/cfnbootstrap/construction.py", line 129, in build
    Contractor(metadata).build(configSets, self)
  File "/usr/lib/python2.7/site-packages/cfnbootstrap/construction.py", line 530, in build
    self.run_config(config, worklog)
  File "/usr/lib/python2.7/site-packages/cfnbootstrap/construction.py", line 542, in run_config
    CloudFormationCarpenter(config, self._auth_config).build(worklog)
  File "/usr/lib/python2.7/site-packages/cfnbootstrap/construction.py", line 260, in build
    changes['commands'] = CommandTool().apply(self._config.commands)
  File "/usr/lib/python2.7/site-packages/cfnbootstrap/command_tool.py", line 117, in apply
    raise ToolError(u"Command %s failed" % name)
ToolError: Command 14_create_symlink failed
2021-02-16 15:07:21,322 [DEBUG] CloudFormation client initialized with endpoint https://cloudformation.us-east-1.amazonaws.com
2021-02-16 15:07:21,322 [DEBUG] Signaling resource PerforceServerInstance in stack perforce-server-awsnva-01 with unique ID i-0c776817bbc062e50 and status FAILURE
```

--- a/templates/PerforceServerSetupTemplate.yaml
+++ b/templates/PerforceServerSetupTemplate.yaml
@@ -260,7 +260,7 @@ Resources:
               ignoreErrors: false
         04_config_install_sdp:
           sources:
-            /tmp: "https://swarm.workshop.perforce.com/projects/perforce-software-sdp/download/downloads/sdp.Unix.tgz"
+            /tmp: "https://swarm.workshop.perforce.com/downloads/guest/perforce_software/sdp/downloads/sdp.Unix.tgz?v=%2314"
           commands:
             01_mv_sdp:
               command: "mv /tmp/sdp /hxdepots"

